tests/backup/backup_license_test.go

```diff
@@ -0,0 +1,140 @@
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			err = CreateSourceAndDestClusters(orgID, "", "", ctx)
+			log.FailOnError(err, fmt.Sprintf("Creating source cluster %s and destination cluster %s", SourceClusterName, destinationClusterName))
+			srcClusterStatus, err = Inst().Backup.GetClusterStatus(orgID, SourceClusterName, ctx)
+			log.FailOnError(err, fmt.Sprintf("Fetching [%s] cluster status", SourceClusterName))
+			dash.VerifyFatal(srcClusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying if [%s] cluster is online", SourceClusterName))
+			destClusterStatus, err = Inst().Backup.GetClusterStatus(orgID, destinationClusterName, ctx)
+			log.FailOnError(err, fmt.Sprintf("Fetching [%s] cluster status", destinationClusterName))
+			dash.VerifyFatal(destClusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying if [%s] cluster is online", destinationClusterName))
+		})
+		Step("Getting the total number of worker nodes in source and destination cluster", func() {
+			log.InfoD("Getting the total number of worker nodes in source and destination cluster")
+			sourceClusterWorkerNodes = node.GetWorkerNodes()
+			log.InfoD("Total number of worker nodes in source cluster are %v", len(sourceClusterWorkerNodes))
+			totalNumberOfWorkerNodes = append(totalNumberOfWorkerNodes, sourceClusterWorkerNodes...)
+			log.InfoD("Switching cluster context to destination cluster")
+			SetDestinationKubeConfig()
+			destinationClusterWorkerNodes = node.GetWorkerNodes()
+			log.InfoD("Total number of worker nodes in destination cluster are %v", len(destinationClusterWorkerNodes))
+			totalNumberOfWorkerNodes = append(totalNumberOfWorkerNodes, destinationClusterWorkerNodes...)
+			log.InfoD("Total number of worker nodes in source and destination cluster are %v", len(totalNumberOfWorkerNodes))
+			log.InfoD("Switching cluster context back to source cluster")
+			err := SetSourceKubeConfig()
+			log.FailOnError(err, "Switching context to source cluster")
+		})
+		Step("Verifying the license count after adding source and destination clusters", func() {
+			log.InfoD("Verifying the license count after adding source and destination clusters")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			licenseInspectResponse, err = Inst().Backup.InspectLicense(ctx, licenseInspectRequestObject)
+			log.FailOnError(err, "Getting the license for organisation %v", orgID)
+			licenseResponseInfoFeatureInfo = licenseInspectResponse.GetLicenseRespInfo().GetFeatureInfo()
+			dash.VerifyFatal(licenseResponseInfoFeatureInfo[0].Consumed, int64(len(totalNumberOfWorkerNodes)), fmt.Sprintf("Verifying the license count when source cluster with %d worker nodes and destination cluster with %d worker nodes are added to backup", len(sourceClusterWorkerNodes), len(destinationClusterWorkerNodes)))
```

Comment:
You could move this part into another function _IF_ you feel like any other tests might be able to use it. Like:
```go
func LicenseConsumedCount(ctx context.Context, orgID string) (int64, error) {
   licenseInspectRequestObject = &api.LicenseInspectRequest{
			OrgId: orgID,
		}
  licenseInspectResponse, err := Inst().Backup.InspectLicense(ctx, licenseInspectRequestObject)
   if err != nil{
			return 0, fmt.Errof("Getting the license for organisation %v.  Err: %v", orgID, err)
  }
			licenseResponseInfoFeatureInfo = licenseInspectResponse.GetLicenseRespInfo().GetFeatureInfo()
 return licenseResponseInfoFeatureInfo[0].Consumed, nil
}
```

Reply:
Used LicenseConsumedCountVerify